tools/spec_coverage/lib/co19.dart

```diff
@@ -0,0 +1,68 @@
+import 'dart:io';
+
+class Co19 {
+  static const languageDirName = "Language";
+  late final TestDir root;
+  late final TestDir language;
+
+  Co19(String path) {
+    Directory dir = Directory(path);
+    if (!dir.existsSync()) {
+      throw Exception("Directory '$path' does not exists");
+    }
+    root = TestDir(dir);
+    for (TestDir td in root.subDirs) {
+      if (td.name == languageDirName) {
+        language = td;
+        break;
+      }
+    }
+  }
+}
+
+class TestDir {
+  final String path;
+  final List<TestDir> subDirs = [];
+  final List<TestFile> tests = [];
+
+  TestDir(Directory root) : path = resolvePath(root.path) {
+    List<FileSystemEntity> entities = root.listSync();
+    for (FileSystemEntity fse in entities) {
+      if (skip(fse)) {
+        continue;
+      }
+      FileStat fs = fse.statSync();
+      if (fs.type == FileSystemEntityType.directory) {
+        subDirs.add(TestDir(Directory(fse.path)));
+      } else if (fs.type == FileSystemEntityType.file) {
+        tests.add(TestFile(fse.path));
```

Comment:
Would it be useful to specify a regular expression matching actual test file names in `Co19` and passing it around such that we can include only test files here? Or perhaps other files (imported libraries and whatnot) could also have an `@assertion` and/or a `@description`?